src/main/java/seedu/financialplanner/storage/LoadData.java

```diff
@@ -36,6 +40,7 @@ public static void load(String filePath) throws FinancialPlannerException {
                 String type = split[0].trim();
                 switch (type) {
                 case "I":
+                    // Fallthrough
```

Comment:
What does this mean? Should I be removed

Reply:
i think its to indicate explicityly that case "I" and "E" have the same body

Reply:
Oh i see